### PR TITLE
style: no error for json views when running eslint

### PR DESCRIPTION
**Pull request by nicogeburek**
#33

### CODE REVIEW

**Comment by nicogeburek** [comment]
#35

**Comment by nicogeburek** [comment]
@petermuessig or @vobu, could you guys have a look at #35 ? I think my PR fixes it for now but we better double check.

File: generators/app/index.js [hunk 1]
```diff
@@ -159,7 +159,7 @@ module.exports = class extends Generator {
                 clearCoverage: "shx rm -rf coverage",
                 karma: "run-s clearCoverage karma-ci",
                 lint: "eslint ./**/webapp/**/*.js && prettier --check ./**/webapp/**/*.{js,xml}",
-                "lint-fix": "eslint ./**/webapp/**/*.js --fix && prettier --write ./**/webapp/**/*.{js,xml}"
+                "lint-fix": "eslint ./**/webapp/**/*.js --fix && prettier --write ./**/webapp/**/*.{js,xml} --no-error-on-unmatched-pattern"
```
Comment (petermuessig) [inline generators/app/index.js:162]: Hmm, but here the parameter looks like being assigned to prettier - was the issue a prettier issue?

Comment (nicogeburek) [reply generators/app/index.js:162]: No, this was the first commit regarding the error message for the linting of JSON Views. Check out the other commit.

**Review by petermuessig — APPROVED** [review]
Thanks for the hint! 😄 

LGTM 👍

**Pull request merged**